environments/extractDiffs.toml

```diff
@@ -203,6 +235,14 @@
+        deployed = 10933898
```

Comment:
Do we need the OSM contracts in this config?

Reply:
nope, probably not. good call!